gcexport.py

```diff
@@ -882,8 +952,10 @@ def main(argv):
         print('Querying list of activities ' + str(total_downloaded + 1) \
               + '..' + str(total_downloaded + num_to_download) \
               + '...', end='')
+        
         logging.info('Activity list URL %s', URL_GC_LIST + urlencode(search_params))
-        result = http_req(URL_GC_LIST + urlencode(search_params))
+        result = http_req_as_string(URL_GC_LIST + urlencode(search_params))
```

Comment:
F-strings again:
```suggestion
        result = http_req_as_string(f'{URL_GC_LIST}{urlencode(search_params)})
```

Reply:
As for Python2 my suggestion should be dropped and keep the original change from the PR